DataFormats/PortableTestObjects/src/classes_def.xml

```diff
@@ -6,7 +6,7 @@
     <field name="y_" comment="[nElements_]"/>
     <field name="z_" comment="[nElements_]"/>
     <field name="id_" comment="[nElements_]"/>
```

Comment:
@ericcano @makortel
I am thinking about renaming `nElements_` to `elements_`, as a parallel to `scalar_`.
What do you think ?

Reply:
I don't have strong feelings either way.

Reply:
As long as the name is clean (and both are), I don't have a strong opinion either.